war/src/main/scss/components/_section.scss

```diff
@@ -36,7 +36,7 @@
 
 .jenkins-section__description {
   // Tweaked margin so that it appears visually centred when placed next to `.jenkins-section__title`
```

Comment:
Does this still apply, and why?

Reply:
Yeah, it makes it a little tighter/more balanced than the previous styling from the previous content PR.